addons/website_sale/static/src/js/cart_service.js

```diff
@@ -273,20 +318,18 @@ export class CartService {
                 isFrontend: true,
                 options,
                 ...additionalData,
-                save: async (comboProductData, selectedComboItems, options) => {
-                    resolve(this._makeRequest({
-                        productTemplateId: productTemplateId,
-                        productId: productId,
-                        quantity: comboProductData.quantity,
-                        // NOTE: no uom since not handled in combo configurator
-                        linked_products: selectedComboItems.map(
-                            (comboItem) => this._serializeComboItem(
-                                comboItem, productTemplateId, comboProductData.quantity
-                            )
-                        ),
-                        shouldRedirectToCart: options.goToCart,
-                        ...additionalData,
-                    }));
+                save: async (comboProductData, selectedComboItems, comboOptions) => {
+                    resolve(this._handleComboSave(
+                        productTemplateId,
+                        productId,
+                        uomId,
+                        shouldShowProductConfigurator,
+                        comboProductData,
+                        selectedComboItems,
+                        comboOptions.goToCart,
+                        comboOptions,
```

Comment:
This one should still be `options` :)
```suggestion
                        options,
```